ci/scripts/pod-to-pod.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -ex
+
+. "$(dirname "$0")/lib/environment.sh"
+
+export BOSH_LOG_LEVEL=debug
+export BOSH_LOG_PATH="${KUBO_DEPLOYMENT_DIR}/bosh.log"
+export DEBUG=1
+
+cp "$PWD/s3-service-creds/service-ci-service-creds.yml" "${KUBO_ENVIRONMENT_DIR}/"
+cp "$PWD/s3-bosh-creds/creds.yml" "${KUBO_ENVIRONMENT_DIR}/"
+
+bosh_ca_cert=$(bosh-cli int "${KUBO_ENVIRONMENT_DIR}/creds.yml" --path=/default_ca/ca)
+client_secret=$(bosh-cli int "${KUBO_ENVIRONMENT_DIR}/creds.yml" --path=/bosh_admin_client_secret)
+
+director_ip=$(bosh-cli int "${KUBO_ENVIRONMENT_DIR}/director.yml" --path="/internal_ip" | xargs echo -n)
+credhub login -u credhub-user -p \
+  "$(bosh-cli int "${KUBO_ENVIRONMENT_DIR}/creds.yml" --path="/credhub_user_password" | xargs echo -n)" \
+  -s "https://${director_ip}:8844" --skip-tls-validation
+
+"${KUBO_DEPLOYMENT_DIR}/bin/set_kubeconfig" "${KUBO_ENVIRONMENT_DIR}" ci-service
+kubectl apply -f "${KUBO_DEPLOYMENT_DIR}/ci/specs/guestbook.yml"
```

Comment:
We can leave is as is for now, as we are destroying the cluster right after this test.